custom/covid/tasks.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+
+from celery.schedules import crontab
+from celery.task import periodic_task
+from dateutil.relativedelta import relativedelta
+
+from casexml.apps.phone.models import SyncLogSQL
+from corehq.apps.formplayer_api.exceptions import FormplayerResponseException
+from corehq.apps.formplayer_api.sync_db import sync_db
+from corehq.apps.users.models import CouchUser
+from corehq.apps.users.util import raw_username
+from corehq.toggles import PRIME_FORMPLAYER_DBS
+from corehq.util.celery_utils import no_result_task
+from dimagi.utils.logging import notify_exception
+from django.conf import settings
+
+SYNC_CUTOFF_HOURS = 8
+
+MIN_CASE_COUNT = 20000
+
+
+@periodic_task(run_every=crontab(minute=0, hour=2), queue=settings.CELERY_PERIODIC_QUEUE)
```

Comment:
Before I say something silly, this runs every 2AM or every 2 hours?

Reply:
2am, I also need to validate this timing based on when the case rules run.

Reply:
Case rules are configurable by domain but USH ones run mostly at 4AM, I believe, although it may be a bit later for some of the further west projects.